app/modules/entities/components/cleanup/serie_cleanup_work.svelte

```diff
@@ -0,0 +1,343 @@
+  export let placeholderTitle: string = null
+  export let allSerieAuthors: SerializedEntity[]
+  export let allSerieParts: WorkWithEditions[]
+  export let selectedLang: WikimediaLanguageCode = null
+  export let nextPlaceholderOrdinalToCreate: number = null
+
+  const { label, serieOrdinalNum } = work
+  let description, pathname, editPathname
+  let nonPlaceholderWork: WorkWithEditions
+  let isPlaceholder = false
+  if (workIsPlaceholder(work)) {
+    ;({ isPlaceholder } = work)
+  } else {
+    ;({ description, pathname, editPathname } = work)
+    nonPlaceholderWork = work
+  }
+
+  const currentWorkAuthorsUris = new Set(getSerieOrWorkExtendedAuthorsUris(work))
+  let [ currentWorkAuthors, authorsSuggestions ] = partition(allSerieAuthors, author => currentWorkAuthorsUris.has(author.uri))
+
+  let showMergeWorkPicker = false
+  let showPlaceholderEditor = false
+
+  const dispatch = createEventDispatcher()
+
+  let merging, flash
+  async function merge (selectedWork: WorkWithEditions) {
+    try {
+      merging = true
+      await mergeEntities(nonPlaceholderWork.uri, selectedWork.uri)
+      dispatch('merged', { from: nonPlaceholderWork, to: selectedWork })
+    } catch (err) {
+      merging = false
+      flash = err
+    }
+  }
+
+  async function addAuthor (author: SerializedEntity) {
+    try {
+      currentWorkAuthors = [ ...currentWorkAuthors, author ]
+      authorsSuggestions = without(authorsSuggestions, author)
+      // TODO: add support for other author roles properties
+      await addClaim(nonPlaceholderWork, 'wdt:P50', author.uri as EntityValue)
+    } catch (err) {
+      flash = err
+    }
+  }
+
+  async function selectOrdinal (e) {
+    const ordinal = e.currentTarget.value
+    if (!isPositiveIntegerString(ordinal)) return
+    try {
+      work = nonPlaceholderWork = await addClaim(nonPlaceholderWork, 'wdt:P1545', ordinal)
+      dispatch('selectOrdinal', nonPlaceholderWork)
+    } catch (err) {
+      flash = err
+    }
+  }
+
+  let placeholderLabel = placeholderTitle
+  let creatingPlaceholder
+  async function createPlaceholder () {
+    try {
+      creatingPlaceholder = true
+      const entity: EntityDraft = {
+        labels: { [selectedLang]: placeholderLabel },
+        claims: {
+          'wdt:P31': [ getDefaultWorkP31FromSerie(serie) ],
+          'wdt:P179': [ serie.uri as EntityValue ],
+          'wdt:P1545': [ serieOrdinalNum.toString() as `${number}` ],
+        },
+      }
+      const createdWork = await createEntity(entity)
+      dispatch('createdPlaceholder', createdWork)
+    } catch (err) {
+      flash = err
+    } finally {
+      creatingPlaceholder = false
+    }
+  }
+
+  function onInputKeyDown (e) {
+    const key = getActionKey(e)
+    if (key === 'esc') showPlaceholderEditor = false
+  }
+
+  $: onChange(placeholderTitle, () => {
+    if (!showPlaceholderEditor) placeholderLabel = placeholderTitle
+  })
+
+  function onNextPlaceholderOrdinalToCreateChange () {
```

Comment:
The last part `Change` made me bug, I tried to integrate with the rest of the function name. Until I realize you meant `onChange` and `NextPlaceholderOrdinalToCreate` right ?
What about following the convention of describing what the function do, and not where/when the function is triggered:

```suggestion
  function createNextPlaceholder () {
```

Ortherwise, I would be for dropping the suffix (as the changing part is already expressed by the prefix `on`)

```suggestion
  function onNextPlaceholderOrdinalToCreate () {
```